qiskit/circuit/gate.py

```diff
@@ -118,16 +117,16 @@ def control(
                 operation.
             ctrl_state: the control state in decimal or as a bitstring
                 (e.g. ``'111'``). If ``None``, use ``2**num_ctrl_qubits-1``.
-            annotated: indicates whether the controlled gate can be implemented
-                as an annotated gate.
+            annotated: indicates whether the controlled gate is implemented
+                as an annotated gate. If ``None``, this is handled as ``False``.
```

Comment:
I have not looked through all of the PR yet, but I am wondering if there is ever a difference between `annotated=False` and `annotated=None`.

Reply:
Yes there is: for all parameterized gates that cannot be synthesized until the parameters are bound. Some examples are: 
- RX or U with more than 1 control
- RXX with any number of controls